### PR TITLE
Take screenshots every second

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
Apart from the screenshots taken in key moments of the e2e testing, this PR makes puppeteer take a screenshot every second. This visual feedback could help us catching more unexpected events.

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]

File: services/web/client/source/class/osparc/auth/ui/RegistrationView.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ qx.Class.define("osparc.auth.ui.RegistrationView", {
 
       const pass1 = new qx.ui.form.PasswordField().set({
         required: true,
-        placeholder: this.tr("Introduce a password")
+        placeholder: this.tr("Type a password")
```
Comment (GitHK) [inline services/web/client/source/class/osparc/auth/ui/RegistrationView.js:59]: I would youse pick a password to inform the user to chose a new one

File: services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js [hunk 2]
```diff
@@ -259,13 +259,13 @@ qx.Class.define("osparc.desktop.preferences.pages.SecurityPage", {
 
       const newTokenKey = new qx.ui.form.TextField();
       newTokenKey.set({
-        placeholder: this.tr("Introduce token key here")
+        placeholder: this.tr("Input token key here")
```
Comment (GitHK) [inline services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js:262]: I would use **Input your**

File: services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js [hunk 3]
```diff
@@ -259,13 +259,13 @@ qx.Class.define("osparc.desktop.preferences.pages.SecurityPage", {
...
       });
       form.add(newTokenKey, this.tr("Key"));
 
       const newTokenSecret = new qx.ui.form.TextField();
       newTokenSecret.set({
-        placeholder: this.tr("Introduce token secret here")
+        placeholder: this.tr("Input token secret here")
```
Comment (GitHK) [inline services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js:268]: I would use **Input your**

**Review by GitHK — COMMENTED** [review]
A few possible improvements

**Pull request merged**